honeybee_vtk/vtkjs_helper.py

```diff
@@ -0,0 +1,213 @@
+import base64
+import json
+import re
+import os
+import shutil
+import sys
+import zipfile
+
+try:
+  import zlib
+  compression = zipfile.ZIP_DEFLATED
+except:
+  compression = zipfile.ZIP_STORED
```

Comment:
When this will be the case?

Reply:
I am not sure. I did not look into this module.